src/relay/op/tensor/transform.cc

```diff
@@ -2248,6 +2248,9 @@ bool GatherNDRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
   Array<IndexExpr> oshape;
   for (size_t i = 1; i < kdim + 1; ++i) oshape.push_back(indices->shape[i]);
   for (size_t i = mdim->value; i < ndim; ++i) oshape.push_back(data->shape[i]);
+  if (oshape.size() == 0) {
```

Comment:
Is this necessary? Looks like we just use a tensor of shape (1,) to replace scalar, and they should be the same?

Reply:
@kevinthesun: I am afraid I didn't get your query clearly here. I am replying based on current understanding, the output should be a tensor of shape 1 in the case if input is 1dim, otherwise we cannot accommodate all the scenarios. Please let me know if it does not answer your query.

Reply:
OK. I think I understand now. https://github.com/apache/incubator-tvm/blob/c71989a9db0d60f6a8077596e2915f9b4537e093/topi/include/topi/transform.h#L1010-L1012 In topi we do the same thing, it's good to align with it.